client/src/components/ValidationLevel.vue

```diff
@@ -1,19 +1,78 @@
 <template>
   <div id="override_level">
-    <Multiselect
-      v-model="value"
+    <multiselect
+      v-model="selected"
       placeholder="Add a validation override level"
-      label="code"
-      track-by="code"
-      :options="options"
+      label="level"
+      track-by="level"
+      :options="filterOptions"
       :multiple="true"
       :taggable="true"
-      :option-height="50"
-      @tag="addLevel"
-    ></Multiselect>
+      :searchable="true"
```

Comment:
Now, we will have all the valid override levels displayed. Users can search using the level or the description.